pkg/chartutil/files_test.go

```diff
@@ -141,3 +141,52 @@ one:
 		t.Fatal("Expected parser error")
 	}
 }
+
+func TestToJson(t *testing.T) {
+	expect := "{\"foo\":\"bar\"}"
```

Comment:
You can simplify the backslash escapes by using backticks to surround the template. Makes it easier to read :)

```
expect := `{"foo":"bar"}`
```